### PR TITLE
[aspnetcore] Fix incorrect logging messages

### PR DESCRIPTION
**Pull request by hauntingEcho**
### PR checklist
 
- [ X ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ X ] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [ X ] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [ X ] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `5.1.x`, `6.0.x`
- [ X ] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request. - @mandrean @frankyjuang @shibayan @Blackclaws

### CODE REVIEW

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java [hunk 1]
```diff
@@ -592,6 +592,7 @@ private void setModelClassModifier() {
     private void setBuildTarget() {
         setCliOption(buildTarget);
         if ("library".equals(buildTarget.getOptValue())) {
+            LOGGER.warn("buildTarget is " + buildTarget.getOptValue() + " so changing default isLibrary to true ");
```
Comment (wing328) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java:595]: Suggestion: 
```
LOGGER.warn("buildTarget is {} so changing default isLibrary to true",  buildTarget.getOptValue());
```

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java [hunk 2]
```diff
@@ -636,7 +637,7 @@ private String determineTemplateVersion(String frameworkVersion) {
 
     private void setUseSwashbuckle() {
         if (isLibrary) {
-            LOGGER.warn("buildTarget is " + buildTarget.getOptValue() + " so changing default isLibrary to false ");
+            LOGGER.warn("isLibrary is true so changing default useSwashbuckle to false ");
```
Comment (wing328) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java:640]: Suggestion: removing the trailing whitespace.

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java (hunk 1 above)
Comment (hauntingEcho) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java]: fixed 4fc83c67c03a26efa4a013e527af0ca56ed11866

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java (hunk 2 above)
Comment (hauntingEcho) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java]: fixed 4fc83c67c03a26efa4a013e527af0ca56ed11866

**Comment by wing328** [comment]
LGTM

**Pull request merged**